doc/Pull Request Acceptance Checklist.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request Acceptance Checklist
+
+## Beagle Project
+ * detailed expanatory Javadoc
+ * No checkstyle violations (errors or warnings)
+ * 100% Code coverage by JUnit Tests
+ * JUnit test check were created by another person and are complete (check everything possible)
```

Comment:
Okay, I see.
Are you able to describe what is meant, so we can replace this misunderstanding sentence?